js/foundation.reveal.js

```diff
@@ -409,7 +414,7 @@ class Reveal extends Plugin {
       var scrollTop = parseInt($("html").css("top"));
 
       if ($('.reveal:visible').length  === 0) {
-        $('html').removeClass('is-reveal-open');
+        removeRevealOpenClasses();
```

Comment:
Please use `this.removeRevealOpenClasses();` (and run the docs and tests before creating a PR)

Reply:
Right. Will do so in 7-8 hours, currently not on one of my machines.